pandas/core/dtypes/dtypes.py

```diff
@@ -60,20 +59,20 @@ class Registry:
     These are tried in order.
     """
     def __init__(self):
-        self.dtypes = []
+        self.dtypes = []  # type: List[ExtensionDtype]
 
-    def register(self, dtype):
+    def register(self, dtype: 'ExtensionDtype') -> None:
         """
         Parameters
         ----------
         dtype : ExtensionDtype
         """
-        if not issubclass(dtype, (PandasExtensionDtype, ExtensionDtype)):
+        if not issubclass(dtype, ExtensionDtype):
```

Comment:
All sublcasses of PandasExtensionDtype are now also ExtensionDtype, so I think this check is not needed anymore.

@TomAugspurger, do you agree?

Reply:
Sounds correct.